packages/react/src/components/SearchField/index.tsx

```diff
@@ -21,6 +21,7 @@ interface SearchFieldProps
   onChange?: (value: string, e: ChangeEvent<HTMLInputElement>) => void;
   hideLabel?: boolean;
   isForm?: boolean;
+  trailingChildren?: React.ReactNode | string;
```

Comment:
`React.ReactNode` already includes string.

```suggestion
  trailingChildren?: React.ReactNode;
```